connector/datadogconnector/config.go

```diff
@@ -63,6 +63,9 @@ type TracesConfig struct {
 	// TraceBuffer specifies the number of Datadog Agent TracerPayloads to buffer before dropping.
 	// The default value is 1000.
 	TraceBuffer int `mapstructure:"trace_buffer"`
+
+	// ResourceAttributesAsContainerTags specifies the list of resource attributes to be used as container tags.
+	ResourceAttributesAsContainerTags []string `mapstructure:"resource_attributes_as_container_tags"`
```

Comment:
Can you share an example configuration, perhaps in the PR description?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/datadogconnector/README.md#configurations and https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/connector/datadogconnector/examples/config.yaml#L25 need to be updated.